### DISCUSSION TITLE
A way forward for Nadir

### DISCUSSION THREAD

**Discussion by Kubius** [goonstation/goonstation#9485] [Future Development Planning]
Design doc 1 (centered around overall design concepts and feature set): https://hackmd.io/@Kubius/SkBk2u3h9
Design doc 2 (centered around acid): https://hackmd.io/@Kubius/SynaXqCTc

Nadir (#7730) has been a big undertaking and I've reached a bit of an impasse on its progress. Currently, the map is feature-complete and capable of being playtested, with gradual improvements occurring both in the core game and the Nadir branch, but without any sort of indication of whether there's staff interest in the map I'm not sure how much more effort I can or should invest.

Main things I'm looking to address here are:

- Is there any sentiment about Nadir among the staff, positive or negative, with regards to including or even testing it? Right now, I'm essentially flying blind with regards to developer opinion.
- I need help to get real large-scale testing off the ground; I've only barely scraped double-digit populace in my tests. I can host an independent server that should accommodate middling double-digit populations tolerably, but it's difficult to determine the map's rough edges without the load that real gameplay provides.
- If sentiment is positive, are there any staff members willing to contribute to the supporting elements that I'm unskilled with? (some examples, though not exhaustive: TGUI work, shuttle docking alterations, new floor sprites, blob/liquid interaction changes)
- Are there any design elements for the map that some consider a "snag" making it seem unsuitable for rotation or lacking in some regard? I have contingencies in mind for a wide variety of requested changes, _most_ of which I can implement without outside assistance.

Some contingencies I had in mind.

- Reintroduction of Miners; the current elevator space is considerably over-provisioned in the interest of being open to the creation of a mining staff room. Another option is also available (mentioning later).
- Inadequacy of the catalytic generator system as a power source; several options available to deal with this. Aside from mechanical alterations, Oshan-style hotspot power could be introduced (exceedingly hazardous but could be some fun danger), or the newly-refurbished nuclear engine currently in progress could be utilized.
- Insufficient map size or bad map layout; I've been tinkering with several fundamental reconstructions of the map with slightly larger overall dimensions, fewer sight lines and more area allocated to dispersed maintenance spaces and pathways.

I can do more, but I need staff feedback and preferably help.

**Comment by ThatFiveGuys** [comment]
It seems odd to have two completely seperate ways of mining done by 3 separate jobs, maybe just have a dedicated mining role (it being in the science department I think would be really cool and fitting)? Would also help with the inevitable issue of people hogging up the harmonic siphon to play with the cool thing.

**Reply by Kubius** [reply]
Introduction of a dedicated "Planetary Assay" job is one of the ideas I had in mind as an addition down the line, responsible for both siphon operation and trench surveying, but *for the moment* I believe Engineering/Sci joint custody over the siphon is an acceptable state, in the interest of getting more hands on the hardware during initial tests. A Planetary Assay split-off would also be well-supported by Engineering receiving the new nuclear engine as a power supply, should that come to pass.

**Comment by Kubius** [comment]
Updated top level post with what I hope is a useful design doc.
